argocd-bootstrap/3-argocd/argocd-basic.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  annotations:
+    example.com/managed-by: gitops
+    example.com/scm-url: git@github.com:redhat-cop/declarative-openshift.git
+  labels:
+    example: basic
+    example.com/component: operators
+    example.com/project: argocd-bootstrap
+  name: example-argocd
+spec:
+  server:
+    route: true
```

Comment:
My guess is that you would not want this to be present in the initial manifest as to add it as part of the exercise